2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -524,7 +524,7 @@
  {
   "input": "I did a whole video on it, it's actually quite famous.",
   "model": "nmt",
-  "translatedText": "Hice un vídeo completo sobre ello, en realidad es bastante famoso.",
+  "translatedText": "Ya hice un vídeo entero sobre ello, de hecho es bastante popular.",
```

I think a "que" would connect this sentences more smoothly:

```suggestion
  "translatedText": "Ya hice un vídeo entero sobre ello, que de hecho es bastante popular.",
```